sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -418,6 +422,12 @@ def encode_to_stream(self, value, stream, nested):
       self.fallback_coder_impl.encode_to_stream(value, stream, nested)
 
   def encode_special_deterministic(self, value, stream):
+    if not self.warn_deterministic_fallback:
+      _LOGGER.warning(
+          "Using fallback deterministic coder for type '%s' in '%s'. ",
+          type(value),
+          self.requires_deterministic_step_label)
+      self.warn_deterministic_fallback = True
```

Comment:
Nit: A True value sounds like it will warn, which is opposite of the semantic here.

Reply:
done, flipped values